llvm/test/Instrumentation/MemorySanitizer/avx512-intrinsics.ll

```diff
@@ -0,0 +1,13602 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt %s -S -mtriple=x86_64-linux-gnu -mattr=+avx512f -passes=msan 2>&1 | FileCheck %s
+;
+; Forked from llvm/test/Instrumentation/MemorySanitizer/avx512-intrinsics.ll
+
+define <8 x double> @test_mask_compress_pd_512(<8 x double> %data, <8 x double> %passthru, i8 %mask) #0 {
+; CHECK-LABEL: @test_mask_compress_pd_512(
```

Comment:
It should be under llvm/test/Instrumentation/MemorySanitizer/X86

also please recommit the test with existing state, including all avx512 intrinsics
probably cloning llvm/test/CodeGen/X86/avx512-intrinsics.ll

Reply:
Done in https://github.com/llvm/llvm-project/pull/124421/commits/56c0267f91967d6db9f9e9f5f93ad63343464230, thanks!